latch_cli/services/init/init.py

```diff
@@ -181,6 +205,12 @@ def init(
             time and the user will not be able to modify it. At any point,
             the user can switch modes by executing `latch dockerfile .` in
             the workflow directory.
+        base_image_type_str: The base image to use for the workflow. Default value
```

Comment:
do these doc strings even matter? is this public API?
anyway, use the more detailed explanation that I wrote

---

also iirc google docstrings recommend dropping the first article so
`Base image` instead of `The base image`